### PR TITLE
Fix several problems in v2 notebooks.

### PR DESCRIPTION
**Pull request by nick863**
# PR into Azure/azureml-examples

## Checklist

I have:

- [x] read and followed the contributing guidelines

## Changes

- Orange juice, bike share and energy notebooks end point will now contain the timestamp as a suffix.
- Orange juice will plot only one grain on the predicted vs actuals plot for simplicity.
- Bike share will demonstrate the data split and removal of columns which results in target leak.

fixes #
Task 2040598
Bug 2040619
Bug 2040624

### CODE REVIEW

File: sdk/python/jobs/automl-standalone-jobs/automl-forecasting-task-bike-share/auto-ml-forecasting-bike-share.ipynb [hunk 1]
```diff
@@ -294,7 +321,7 @@
     "\n",
     "**Target column** is what we want to forecast.\n",
     "\n",
-    "**Time column** is the time axis along which to predict."
+    "**Time column** is the time axis along which to predict (we have defined it when we have split the data)."
```
Comment (vlbejan) [inline sdk/python/jobs/automl-standalone-jobs/automl-forecasting-task-bike-share/auto-ml-forecasting-bike-share.ipynb:324]: nit: we defined it when we split the data

Comment (nick863) [reply sdk/python/jobs/automl-standalone-jobs/automl-forecasting-task-bike-share/auto-ml-forecasting-bike-share.ipynb]: Fixed.

File: sdk/python/jobs/automl-standalone-jobs/automl-forecasting-task-bike-share/auto-ml-forecasting-bike-share.ipynb [hunk 2]
```diff
@@ -128,8 +128,39 @@
    "source": [
     "## 2. Data\n",
     "\n",
-    "We will use Bike Share Data for model training. The data is stored in a tabular format.\n",
+    "We will use Bike Share Data for model training. The data is stored in a csv file. To use these data for training, we will split the data into train and test sets and convert training set to MLTable format. We will define time column name to split the data by timestamp.<br>\n",
+    "> <font color=red>**Important!**</font> In this data set we have two column _causal_ and _registered_, sum of which results in our target column _cnt_. We have to remove these columns to avoid target leak. Please make sure to remove all columns, value of which is unknown in future before training a model."
```
[the remark this answers is not in the document]
Comment (nick863) [reply sdk/python/jobs/automl-standalone-jobs/automl-forecasting-task-bike-share/auto-ml-forecasting-bike-share.ipynb]: Fixed.

File: sdk/python/jobs/automl-standalone-jobs/automl-forecasting-orange-juice-sales/automl-forecasting-orange-juice-sales-mlflow.ipynb [hunk 3]
```diff
@@ -1009,7 +1014,7 @@
       "metadata": {},
       "source": [
         "### Forecast versus actuals plot.\n",
-        "We will join historical data with the predictions to plot predictions and actuals on a time series plot."
+        "We will join historical data with the predictions to plot predictions and actuals on a time series plot. For simplicity, we will select the series for store 2 and \"dominicks\" brand."
```
[the remark this answers is not in the document]
Comment (nick863) [reply sdk/python/jobs/automl-standalone-jobs/automl-forecasting-orange-juice-sales/automl-forecasting-orange-juice-sales-mlflow.ipynb]: Fixed

**Review by vlbejan — APPROVED** [review]

**Review by jialiu103 — APPROVED** [review]

**Pull request merged**